src/components/LinterMessage/index.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Alert } from 'react-bootstrap';
+
+import styles from './styles.module.scss';
+import { LinterMessage as LinterMessageType } from '../../linter';
+
+export type PublicProps = {
+  description: LinterMessageType['description'];
+  message: LinterMessageType['message'];
+  type: LinterMessageType['type'];
+};
+
+const getAlertVariant = (type: PublicProps['type']) => {
+  switch (type) {
+    case 'error':
+      return 'danger';
+    case 'warning':
+      return 'warning';
+    default:
+    case 'notice':
+      return 'secondary';
+  }
+};
+
+const urlPattern = /^https?:\/\//;
+
+const renderDescriptionPart = (part: string) => {
+  // Split a description into words to intercept and replace
+  // URLs with JSX links.
+  return part
+    .trim()
+    .split(' ')
```

Comment:
Would it make sense to split on `\s` instead?

Reply:
Whoa, `.split(/(\s)/)` magically adds the space back! I didn't know that.

> If separator is a regular expression that contains capturing parentheses, then each time separator is matched, the results (including any undefined results) of the capturing parentheses are spliced into the output array.